### DISCUSSION TITLE
adding lua commands I dont want mapped

### DISCUSSION THREAD

**Discussion by fhill2** [mrjones2014/legendary.nvim#147] [Q&A] [answered]
First of all, thank you for an awesome plugin. It has been very useful to me.

Secondly, I have some custom lua functions that I would like to show in the Legendary display.
I don't use them very often so I don't want mappings for them.
They are mostly my own snippets of code without associated nvim commands for them.

I tried adding them to Legendary:
___
#### Keymaps
ERROR: needs lhs for it to be added
```lua 

keymaps = {
{'', require"util/custom".func, description = "My Own Function"  }
}
```

#### Commands
ERROR: command doesnt start with a capital letter.

```lua
commands = {
    { '<cmd>lua require"plugin.sniprun.repl".toggle()<cr>', require"plugin.sniprun.repl".toggle, description = 'sniprun REPL toggle' }, 
}
```
Can I add these lua functions to Legendary without mapping them or creating nvim commands that call the lua functions?

**Comment by mrjones2014** [comment, marked as answer]
Just give the command a name:

```lua
commands = {
    { ':SnipRunRepl', require"plugin.sniprun.repl".toggle, description = 'sniprun REPL toggle' }, 
}
```

**Reply by fhill2** [reply]
Ah, I was assumming the command had to exist for Legendary to map it. 

Thankyou, works!

**Reply by mrjones2014** [reply]
Nope, Legendary can handle creating the commands and keymaps for you!
